.circleci/config.yml

```diff
@@ -0,0 +1,203 @@
+          command: |
+            # Get the longuest line width (ignoring release links)
+            test $(cat CHANGELOG.md | grep -Ev "^\[.*\]: https://github.com/openfun" | wc -L) -le 80
+
+  # Build the Docker image used in development
+  build-docker:
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build development image
+          command: make build
+      - run:
+          name: Check built image availability
+          command: docker images "potsie:*"
+
+  # Compile Jsonnet sources
+  compile:
+    machine:
+      image: ubuntu-2004:202104-01
+      docker_layer_caching: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - run:
+          name: Build development image
+          command: make build
+      - run:
+          name: Compile sources
+          command: make compile
+      - persist_to_workspace:
+          root: .
+          paths:
+            - var/
+
+  # Lint Jsonnet sources
+  lint:
+    machine:
+      image: ubuntu-2004:202104-01
+      docker_layer_caching: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - run:
+          name: Build production image
+          command: make build
+      - run:
+          name: Format sources
+          command: bin/jsonnetfmt --test $(find src/ -type f -name '*.jsonnet')
+      - run:
+          name: Lint sources
+          command: make lint
+
+  # Make a new github release
+  release:
+    docker:
+      - image: circleci/buildpack-deps:stretch-scm
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/fun
+      - run:
+          name: Install gh CLI
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | \
+              sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | \
+              sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            sudo apt update
+            sudo apt install gh
+      - run:
+          name: Define release reference
+          command: |
+            echo 'RELEASE=$([[ -z "$CIRCLE_TAG" ]] && echo "latest" || echo "${CIRCLE_TAG}")' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Build release archive
+          command: |
+            mkdir releases
+            tar cvzf releases/postie-${RELEASE}.tgz -C ~/fun var/lib/grafana
+      - run:
+          name: Create release on GitHub
+          command: |
+            gh config set prompt disabled
+            if [[ "${RELEASE}" == "latest" ]]; then
+              gh release delete -y "${RELEASE}" || true;
+              git push origin ":${RELEASE}";
```

Comment:
What do you push here ?

Reply:
I'm deleting the remote `latest` tag so that it points to the latest commit and not an old one.

Reply:
The `gh release delete` command does not remove release-attached tag...